src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
 public class Application {
+    public static final int BALLS_NUMBER = 3;
```

Comment:
Let's also pay attention to our access modifiers. It's better to make it as less visible as possible. So, I'd recommend making it private here